README.md

```diff
@@ -50,10 +52,16 @@ $ npx jest-codemods --help
 
     Examples:   npx jest-codemods src
                 npx jest-codemods src/**/*.test.js
+                npx jest-codemods src/**/*.test.js --parser=babel --skipImportDetection=0 --standaloneMode
 
     Options:
-      -f, --force       Bypass Git safety checks and force codemods to run
-      -d, --dry         Dry run (no changes are made to files)
+      -f, --force             Bypass Git safety checks and force codemods to run
+      -d, --dry               Dry run (no changes are made to files)
+      --parser                The parser to use (babel, flow, ts, tsx)
+      --transformer           The transformer to use (ava,     chai-assert, chai-should, expect-js, expect, jasmine-globals,     jasmine-this, mocha, should, tape)
+      --skipImportDetection   Keep using the global object for     assertions
+      --standaloneMode        Use explicit require calls instead of     globals
```

Comment:
`Use explicit require calls instead of globals` -> `Use explicit require calls to Jest instead of globals (not recommended unless you run the tests in a browser)`